articles/connections/passwordless/ios-magic-link.md

```diff
@@ -29,6 +29,10 @@ iOS needs to know which domains your application handles. To configure this:
 2. Find the *Associated Domains* section, and move the slider (located near the top right) so that it displays **On**. This enables the use of Associated Domains.
 3. Click on the **plus sign** to add your Auth0 Application's domain. You'll need to use the following format: `applinks:${account.namespace}`
 
+::: note
+If you use custom domains, replace `account.namespace` with your custom domain.
+:::
```

Comment:
Could probably break this note up into a partial and include it in each

Reply:
Yeah - I just realized that. Thanks. I'll fix it.